sn_cli/src/subcommands/files/mod.rs

```diff
@@ -235,13 +242,67 @@ async fn upload_files(
         batch_size,
     };
 
+    // Max amount of sequential payment failures before we bail
+    let max_sequential_payment_fails = 3;
```

Comment:
this shall be the `max_retries` opt instead ?
if the `payment` attempt has to be undertaken for 3 times, it's acutally `min`, and better make it as a const ?